charms/jupyter-controller/config.yaml

```diff
@@ -1,8 +1,20 @@
 options:
+  cluster-domain:
+    type: string
+    default: cluster.local
+    description: Specifies the base domain name of the Kubernetes cluster, which is used in culling to correctly resolve and address the services and resources within the cluster.
+  cull-idle-time:
+    type: int
+    default: 1440
+    description: The amount of time (in seconds) that a Jupyter notebook server can remain idle before it is automatically shut down.
```

Comment:
Should we add a comment somewhere that this is now configurable? Not sure if this is the perfect place https://github.com/canonical/bundle-kubeflow/issues/913 but yeah I 'm thinking of release notes.